### ISSUE TITLE
Get 1.0.0 out

### ISSUE THREAD

**Issue by paulmillr** [paulmillr/es6-shim#404]
Thoughts on this? We almost have ES2016 at this point.

**Comment by ljharb** [comment]
I'd hoped to make it more modular first so that v1.0.0 would collect a bunch of breaking changes at once, but that's OK to do in a v2.0.0.

I would like to drop bower and component support as part of v1 at least, as well as start with a fixed tagging scheme.

**Comment by blacksonic** [comment]
It would be good, because on projects like Angular 2 that allow minor version upgrades it can be upgraded. Now it is not possible because of pre 1.0 version.

**Comment by ljharb** [comment]
@blacksonic it should still work fine if using `^`, which for < 1.0 releases upgrades the patch version (which is the only nonbreaking thing that can be updated pre-1.0)

**Comment by blacksonic** [comment]
Yes i just realized that minor version changes pre-1.0 are actualling breaking changes
